# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/base/NetworkConnectivityService.cpp
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/base/NetworkConnectivityService.cpp
--- a/netwerk/base/NetworkConnectivityService.cpp	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/base/NetworkConnectivityService.cpp	Tue Feb 12 16:08:25 2019 +0000
@@ -237,13 +237,13 @@
   nsresult rv;
   mIPv4Channel = SetupIPCheckChannel(/* ipv4 = */ true);
   if (mIPv4Channel) {
-    rv = mIPv4Channel->AsyncOpen2(this);
+    rv = mIPv4Channel->AsyncOpen(this);
     NS_ENSURE_SUCCESS(rv, rv);
   }
 
   mIPv6Channel = SetupIPCheckChannel(/* ipv4 = */ false);
   if (mIPv6Channel) {
-    rv = mIPv6Channel->AsyncOpen2(this);
+    rv = mIPv6Channel->AsyncOpen(this);
     NS_ENSURE_SUCCESS(rv, rv);
   }
 